packages/core/content-type-builder/admin/src/components/FormModalHeader/index.js

```diff
@@ -107,6 +112,7 @@ const FormModalHeader = ({
   return (
     <ModalHeader>
       <Flex gap={3}>
+        {showBackLink && (<Link aria-label="Back" startIcon={<ArrowLeft />} onClick={() => onOpenModalAddField({ forTarget, targetUid })} to="#back" />)}
```

Comment:
@joshuaellis This needs translation :) Could you add that as a follow-up? 🙏🏼

Reply:
@gu-stav I can add them if @joshuaellis is busy. Just let me know.

Reply:
be awesome if you could @nitinmadelyn

Reply:
@gu-stav @joshuaellis Open the PR #17271